workflow-templates/im-test-k6-manual.yml

```
@@ -0,0 +1,78 @@
+name: Manual trigger K6 test
+
+on: 
+  workflow_dispatch:
+    inputs:
+      envToTest:
+        description: 'Environment to run test against [dev, qa, stage]'   
+        required: true
+        default: 'dev'
+      testFolderRoot:
+        description: 'Test folder root directory from root of repo'
+        required: true
+        default: "" # TODO: Verify this is the correct location of your tests.  IE: tests/k6
+      testFile:
+        description: 'K6 test file to run'     
+        required: true
+        default: '' # TODO: Default test file  IE: check-health-simple.js
+      k6_extra_params:
+        description: 'K6 command extra paramaters'
+        required: false
+        default: '' # TODO: Set default paramaters to run on every K6 test. IE. --vus 1 -i 1 --insecure-skip-tls-verify -out JSON="./k6-output.json" --out influxdb="http://influxdb:8086/k6" --config "/scripts/config/dev-config.json"
+      k6_version:
+        description: 'K6 version'
+        required: true
+        default: '0.33.0' # TODO: Set default verion of k6
```

In this case, I probably would not provide a value, I'd leave it up to the user so the workflow does not run correctly until someone explicitly puts a value in.  That's how most of the templates are set up.
```suggestion
        default: '' # TODO: Set default version of k6
```